docs/user-guide/station-credential-management.md

```diff
@@ -0,0 +1,61 @@
+# LoRa Basics™ Station credentials management in 'Azure IoT Edge LoRaWAN Starter Kit'
+
+Starting with 'Azure IoT Edge LoRaWAN Starter Kit' v2.0.0, the LoRaWan Network Server runs a WebSocket endpoint compatible with [LNS Protocol](https://doc.sm.tc/station/tcproto.html) from LoRa Basics™ Station.
+
+As it's possible to read in [official LoRa Basics™ Station documentation](https://doc.sm.tc/station/credentials.html) a Basics™ Station client needs some credentials to establish a secure connection to LNS compatible endpoints.
+
+## Server Authentication
+
+### Importing certificate for server authentication in LoRaWan Network Server module
+
+LoRaWan Network Server IoT Edge module allows to import, from disk, a certificate for server authentication in 'pkcs12' format (.pfx).
+
+Two environment variables need to be set for making this happen:
```

Comment:
Didn't we set this to be optional and we would load at a default location?

Reply:
No, optional load is only for Basics Station client module as of now